source/wp-content/themes/wporg-themes-2024/inc/block-bindings.php

```diff
@@ -78,7 +78,7 @@ function get_meta_block_value( $args, $block ) {
 			return sprintf(
 				'<a href="%s">%s</a>',
 				esc_url( get_support_url( $theme->slug . '/reviews/' ) ),
-				__( 'See all reviews', 'wporg-themes' )
+				__( 'See all<span class="screen-reader-text"> reviews</span>', 'wporg-themes' )
```

Comment:
I updated this to bring back "reviews" for screen reader users, so navigating by link isn't a mystery :)